torch/_inductor/ir.py

```diff
@@ -5128,6 +5133,11 @@ def realize_input(cls, x):  # type: ignore[no-untyped-def]
                 except NotImplementedError:
                     pass
         if isinstance(x, StorageBox):
+            if isinstance(x.data, ShapeAsConstantBuffer):
```

Comment:
do you know why we're mapping ShapeAsConstantBuffer with StorageBox ? that seems weird

Reply:
It happens here. I can prevent wrapping of symints here. Is that more intuitive?

https://github.com/pytorch/pytorch/blob/main/torch/_inductor/lowering.py#L6779

Reply:
Yes, I think so.

Reply:
Ok, updated.